src/Refinement/RefinedTriangulations.jl

```diff
@@ -0,0 +1,126 @@
+
+
+struct RefinedTriangulation{Dc,Dp,A<:Triangulation{Dc,Dp},B<:RefinedDiscreteModel} <: Triangulation{Dc,Dp}
+  trian::A
+  model::B
+end
+
+function get_refined_model(t::RefinedTriangulation)
+  return t.model
+end
+
+# Wrap Triangulation API
+function Geometry.get_background_model(t::RefinedTriangulation)
+  get_background_model(t.trian)
+end
+
+function Geometry.get_grid(t::RefinedTriangulation)
+  get_grid(t.trian)
+end
+
+function Geometry.get_glue(t::RefinedTriangulation,::Val{d}) where d
+  get_glue(t.trian,Val(d))
+end
+
+function Base.view(t::RefinedTriangulation,ids::AbstractArray)
+  v = view(t.trian,ids)
+  return RefinedTriangulation(v,t.model)
+end
+
+# Wrap constructors for RefinedDiscreteModel
+function Geometry.Triangulation(
+  ::Type{ReferenceFE{d}},model::RefinedDiscreteModel,filter::AbstractArray) where d
+  
+  trian = Triangulation(ReferenceFE{d},get_model(model),filter)
+  return RefinedTriangulation(trian,model)
+end
+
+function Geometry.Triangulation(
+  ::Type{ReferenceFE{d}},model::RefinedDiscreteModel,labels::FaceLabeling;kwargs...) where d
+  trian = Triangulation(ReferenceFE{d},get_model(model),labels;kwargs...)
+  return RefinedTriangulation(trian,model)
+end
+
+function Geometry.Triangulation(trian::RefinedTriangulation,args...;kwargs...)
```

Comment:
Not sure why this constructor is needed. I am not saying it is not required, I just do not see why it is there (most probable I am missing something). 

For example, isn't `Triangulation(trian.trian)===trian.trian` ?

Reply:
isn't a bit strange that `args` and `kwargs` are not used at all?

Reply:
Resolved by [26a64d3](https://github.com/gridap/Gridap.jl/pull/838/commits/26a64d30ea950f576117ead3de9255f5e2ce5ad4)